### ISSUE TITLE
Support `src` attributes on tags

### ISSUE THREAD

**Issue by Akryum** [Akryum/vue-meteor#128]
Example:

```html
<template src="./template.html"></template>
<style src="./style.css"></style>
<script src="./script.js"></script>
```

**Comment by AsafAgranat** [comment]
Can external files not be defined in the script itself? It would make it more sensible than having empty template and style tags. 
Like template: './template.html'
And style: './style.scss'

**Comment by Akryum** [comment]
This is to get the compiler in line with the [specs](http://vue-loader.vuejs.org/en/start/spec.html) as much as possible.

**Comment by WonderToys** [comment]
I see you added this as an Important Feature for RC. I just want to add a second to that as I feel this is an extremely important feature. On any non-trival app, .vue files can become rather big. Being able to split styles out into separate files would be such a huge win.

**Comment by Akryum** [comment]
Implemented in `0.9.0` release.
